package.json

```diff
@@ -219,7 +219,7 @@
       },
       {
         "command": "marquee.snippet.insertEditor",
-        "title": "Insert from Snippets",
+        "title": "Insert from Clipboard",
```

Comment:
I wonder if this might confuse people thinking they actually insert something from their system clipboard.

Reply:
`Insert from Marquee Clipboard` or `Insert from Clipboard Item`